### DISCUSSION TITLE
Transaction issue

### DISCUSSION THREAD

**Discussion by mickeyslayy** [blockscout/blockscout#11370] [Support]
I sent my 35 coins to a wrong user 0x46ed16a47f1f862736be7849666b06216b248e10 
This is the id now I'm trying to contact him so they can send the coin back coz I need to pay my rent with that coin money. There is no option I can contact him please help.

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
